### PR TITLE
CP-266 : Ignore errors and fix source map

### PR DESCRIPTION
**Pull request by LoganPB**
## 📝 What is this PR

This PR fixes the source map generation for Sentry.
It ignores two errors that are spamming Sentry.
Upgrade the SDK of the Sentry Javascript library.

-   [Specs](https://phantombuster.atlassian.net/browse/CP-266)

## 🏭 How does it work

On the sentry-release.yml, there was an error, and the wrong build was done.
To ignore the message, Sentry.init has a parameter "ignoreErrors", so I added the two errors two filters.

## 🗒 Notes

**Something you need to inform your reviewers about**

-   [ ] This change requires a documentation update
-   [ ] This change requires a unit tests update

## 🧪 QA Process

**Details a QA process to the reviewers**

### ✅ Positive testing (should succeed)

-   When the release will be done, we have to check if the source maps are correctly uploaded on Sentry and if the logs don't appear again.

## 👌 To check

-   [ ] There's no regression
-   [ ] Coding guidelines are followed ([Contributing](https://github.com/phantombuster/engineering-docs/blob/master/contributing.md), [Testing](https://github.com/phantombuster/engineering-docs/blob/master/testing.md))

### CODE REVIEW

File: src/shared/sentry.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export function initSentry() {
 	Sentry.init({
 		dsn: "https://a4bfab3486a647ea94cab5580874e008@o303567.ingest.sentry.io/6698508",
 		release: version,
+		ignoreErrors: ["Non-Error exception captured", "ResizeObserver loop limit exceeded"],
```
Comment (Oxygen66) [inline src/shared/sentry.ts:9]: Can you document this line please ?

**Review by Oxygen66 — CHANGES_REQUESTED** [review]

**Review by RiahiKarim — DISMISSED** [review]
LGTM 💯 .

It seems that `ResizeObserver - loop limit exceeded` errors can be [safely ignored](https://stackoverflow.com/a/50387233) but I am not sure about the [Non-Error Exception events ](https://docs.sentry.io/platforms/javascript/troubleshooting/#events-with-non-error-exception), we might be doing something wrong.

In any case, I think it is a good idea to merge your PR to ignore these two errors and maybe look further for the Non-Error Exception events.

**Review by Hatscat — APPROVED** [review]
LGTM

**Review by RiahiKarim — APPROVED** [review]
LGTM

**Review by Oxygen66 — APPROVED** [review]
LGTM 👍

**Pull request merged**